rllib/utils/serialization.py

```diff
@@ -0,0 +1,149 @@
+import base64
+import gym
+import io
+import numpy as np
+from typing import Dict
+import zlib
+
+
+def serialize_ndarray(d: np.ndarray) -> str:
```

Comment:
can we call this `array` instead of `d`? ok in tests, but let's not be too cryptic here!

Reply:
Agreed, we should never use single-char variable names.

Reply:
bad habit from working with other languages. I agree function signature shouldn't use these super short-hands.
one funny thing is that I don't know a good word to use instead of "s" for the deserialize_ndarray() ...

Reply:
think this is fine. especially for complex and math-heavy functionality, imo it can even be an advantage to have concise notation. depends on context, of course.